R/dependencies.R

```diff
@@ -536,25 +540,34 @@ renv_dependencies_discover_description <- function(path,
       sprintf("Config/renv/profiles/%s/dependencies", profile)
 
     fields <- c(fields, "Suggests", profile_field)
-    type <- renv_description_type(desc = dcf)
+    proj_desc <- TRUE
 
   } else {
-    type <- "unknown"
+    proj_desc <- FALSE
   }
 
   data <- map(
     fields,
     renv_dependencies_discover_description_impl,
     dcf  = dcf,
     path = path,
-    type = type
+    proj_desc = proj_desc
   )
 
+  # Infer a dependency on roxygen2 and devtools if RoxygenNote present
+  if (!is.null(dcf$RoxygenNote)) {
+    data <- c(
+      data,
+      list(renv_dependencies_list(path, c("roxygen2", "devtools"), dev = TRUE))
```

Comment:
IMO this only needs to add roxygen2, not devtools, but that would be a behaviour change.